ogi/components/OgImage/blog.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import type { CSSProperties } from 'vue';
+
+// inherited attrs can mess up the satori parser
+defineOptions({
+  inheritAttrs: false,
+});
+
+defineProps<{
+  title: string;
+  image: string;
+}>();
+
+const cover: CSSProperties = {
+  objectFit: 'cover',
+  objectPosition: 'center',
```

Comment:
have you tried `object-cover object-center`?

Reply:
let met check, hopefully Satori support that

Reply:
satori still not support that class